include/libdnf/rpm/package_query.hpp

```diff
@@ -383,7 +383,7 @@ class PackageQuery : public PackageSet {
     ///
     /// @replaces libdnf/sack/query.hpp:method:addFilter(int keyname, int cmp_type, int match) - cmp_type = HY_PKG_LATEST
     /// @replaces libdnf/sack/query.hpp:method:addFilter(int keyname, int cmp_type, int match) - cmp_type = HY_PKG_LATEST_PER_ARCH
-    PackageQuery & filter_latest(int limit = 1);
+    PackageQuery & filter_latest_evr(int limit = 1);
```

Comment:
Are you sure that the name `filter_latest_evr` is more informative? It replaces `HY_PKG_LATEST_PER_ARCH` and obsoletes `HY_PKG_LATEST`.

Reply:
Let's keep it this way to give users a hint that it's **not** the latest package but that it works with groups of nevras.
A more correct method name would be something like filter_latest_evr_per_na(), but that would be probably too much :)